package.json

```diff
@@ -161,6 +162,7 @@
     "ramda": "^0.21.0",
     "react": "^16.13.1",
     "react-aria-modal": "^3.0.1",
+    "react-circular-progressbar": "^2.0.3",
```

Comment:
Why was this added?

Reply:
We are not sure about all the changes in this file. We'll look into it